server/routes/posts.js

```diff
@@ -0,0 +1,116 @@
+var express = require("express");
+var router = express.Router();
+const Post = require("../models/post");
+const User = require("../models/user");
+
```

Comment:
Add a route GET post/postId. It should return the comments and likes for a post. To be triggered when user clicks on a post